common/TelemetryEvents/DeveloperIdHelper.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft Corporation and Contributors
+// Licensed under the MIT license.
+
+using System;
+using System.Security.Cryptography;
+using System.Text;
+using Microsoft.Windows.DevHome.SDK;
+
+namespace DevHome.Common.TelemetryEvents;
+public static class DeveloperIdHelper
+{
+    public static string GetHashedDeveloperId(string providerName, IDeveloperId devId)
+    {
+        // TODO: Instead of LoginId, hash a globally unique id of DeveloperId (like url)
+        // https://github.com/microsoft/devhome/issues/611
+        using var hasher = SHA256.Create();
+        var loginIdBytes = Encoding.ASCII.GetBytes(devId.LoginId());
+        var hashedLoginId = hasher.ComputeHash(loginIdBytes);

```

Comment:
Not blocking feedback, but would it be a little more readable the hash were truncated? We dont actually need the hash to be matchable to anything, just a quasi-unique identifier obfuscating the real developer id, right? In that case IMO it would be nice to shorten it since a sha 256 hash can be quite a long string.